pa11y/mappings/common/createCommonMappings.ts

```diff
@@ -0,0 +1,90 @@
+import { Options } from "wiremock-rest-client/dist/model/options.model";
+import wiremock from "../../lib/general";
+
+export default (baseUri?: string, options?: Options) => {
+  // Mapping for covers.
+  wiremock(baseUri, options).mappings.createMapping({
+    request: {
+      method: "GET",
+      urlPattern: "/api/v2/covers.*",
+    },
+    response: {
+      jsonBody: [
+        {
+          id: "870970-basis:134693959",
+          type: "pid",
+          imageUrls: {
+            small: {
+              url: "https://picsum.photos/98/160",
+              format: "jpeg",
+              size: "small",
+            },
+            large: {
+              url: "https://picsum.photos/142/172",
```

Comment:
Please document these picsum urls.

Why are these specific urls relevant in the current context? Does the url arguments refer to any expected sizes? Was picsum chosen because it supports features we need, do we use it elsewhere in the project or perhaps it is just random? :smile:

Reply:
I'll document it.
The equivalent to this in Cypress is done via fixtures.
This was done as an easy way to get a picture dummy with proper rights.
Otherwise one could look into if Wiremock supports bas64 encoded mocks.
But I prioritized my time.